perf/micro/immediate-scheduler/observable/from-array.js

```diff
@@ -8,7 +8,7 @@ module.exports = function fromArray(suite) {
     }
 
     var oldFromArrayWithImmediateScheduler = RxOld.Observable.fromArray(args, RxOld.Scheduler.immediate);
-    var newFromArrayWithImmediateScheduler = RxNew.Observable.fromArray(args);
+    var newFromArrayWithImmediateScheduler = RxNew.Observable.from(args);
```

Comment:
Ah, I see, it's still faster than Rx2, but it's half the speed of just using fromArray. Amazing that one `isArray` check would make such a difference.